packages/graphql/src/codegen/watch.ts

```diff
@@ -0,0 +1,74 @@
+import fs from "fs";
+import Path from "path";
+import md5 from "md5";
+import minimatch from "minimatch";
+
+/**
+ * A function to watch a given path and call a handler whenever a change is made.
+ * @param path A path to watch for changes. The path can be a file, directory,
+ *   or glob. Globs are matched using [minimatch](https://github.com/isaacs/minimatch).
+ * @param handler The handler to call each time a file matching the path is
+ *   changed.
+ */
+export function watch(path: string, handler: (filename: string) => void): void {
```

Comment:
nice, I wonder if we could use `gaze` for this: https://www.npmjs.com/package/gaze

Reply:
Captured this is CU as a todo for possible improvement. https://app.clickup.com/t/2kxz33f